### ISSUE TITLE
[qt] Binary format doesn't compile

### ISSUE THREAD

**Issue by MartinDelille** [OpenAPITools/openapi-generator#3421]
##### Description

I tried to describe a binary format file download as describe here: https://swagger.io/docs/specification/describing-responses/

Unfortunately, the generated code doesn't compile.

##### openapi-generator version

4.0.3

##### OpenAPI declaration file content or url

```yaml
openapi: 3.0.1
servers:
  - url: 'https://api.myapp.com/v1'
info:
  description: This is myapp.com API.
  version: 1.0.0
  title: myapp.com
paths:
  /picture/{pictureId}:
    get:
      operationId: findPictureById
      parameters:
        - name: pictureId
          in: path
          schema:
            type: string
          required: true
      responses:
        '200':
          description: A picture file
          content:
            image/png:
              schema:
                type: string
                format: binary
```

##### Command line used for generation

```
openapi-generator generate -i api.yaml -g cpp-qt5-client
```

##### Steps to reproduce

Generate the upper spec with the given command and try to compile.

##### Related issues/PRs

No

##### Suggest a fix/enhancement

I'll be happy to help to fix this.

**Comment by macjohnny** [comment]
@MartinDelille a PR would be very welcome!

**Comment by MartinDelille** [comment]
@macjohnny Would you have another codegen I could inspire myselft to do it?

**Comment by macjohnny** [comment]
maybe the `typescript-angular` or `typescript-rxjs`?

**Comment by etherealjoy** [comment]
general issue for all C++ generators :(

**Comment by etherealjoy** [comment]
solved in #3853
